### ISSUE TITLE
Danish locale

### ISSUE THREAD

**Issue by Morsby** [ownaginatious/fbchat-archive-parser#34]
Hi,

Thanks for making this tool!

Just attempted to use this on data downloaded in the Danish locale. I get the following error:

```
Discovered chat thread with [XX, YY]...
Unexpected time format in "3. december 2015 kl. 10:29 UTC+01".  If you downloaded your 
Facebook data in a language other than English,  then it's possible support may need to
be  added to this tool.

Please report this as a bug on the associated GitHub page and it will be fixed promptly
```

The date format ("3. december 2015 kl. 10:29 UTC+01") translates to "3rd December 2015 at. 10:29 UTC+01" (as you might've been able to guess).

**Comment by Morsby** [comment]
Sorry - I think I fixed this myself using this [comment](https://github.com/ownaginatious/fbchat-archive-parser/issues/27#issuecomment-275176448) and adding Danish locale (though I don't know how it handled the timezone (UTC+01).

**Comment by ownaginatious** [comment]
Thanks for the example. I added proper support for Danish 👍

There are only a small set of possible timezone formats, and they actually change independently from the language settings depending on locale, so you don't need to worry about it.

Danish support is now available on PyPI as of version `0.8.post34` :)
